lib/main.dart

```diff
@@ -1,125 +1,155 @@
+                  child: Container(
+                    height: 60,
+                    decoration: BoxDecoration(
+                      gradient: LinearGradient(
+                        colors: [
+                          Colors.transparent,
+                          Colors.black.withOpacity(0.7),
+                        ],
+                        begin: Alignment.topCenter,
+                        end: Alignment.bottomCenter,
+                      ),
+                    ),
+                    child: Center(
+                      child: Text(
+                        movie.title,
+                        style: UIConstants.title,
+                      ),
+                    ),
+                  ),
+                ),
+              ]),
+              Padding(
+                padding: const EdgeInsets.all(20.0),
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: <Widget>[
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceAround,
+                      children: [
+                        Container(
+                          decoration: BoxDecoration(
+                            borderRadius: BorderRadius.circular(5),
+                            border: Border.all(color: Colors.white70),
+                          ),
+                          child: const Padding(
+                            padding: UIConstants.paddingPlayButton,
+                            child: Row(
+                              children: [
+                                Icon(
+                                  Icons.play_arrow,
+                                  color: Colors.white70,
+                                ),
+                                Text('Play trailer', style: UIConstants.text),
+                              ],
+                            ),
+                          ),
+                        ),
+                        Row(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          children: [
+                            IconButton.outlined(
+                                onPressed: _incrementCounter,
+                                icon: const Icon(Icons.favorite,
+                                    color: Colors.white70)),
+                            Text(
+                              '$_counter',
+                              style: UIConstants.text,
+                            ),
+                          ],
+                        ),
+                      ],
+                    ),
+                    UIConstants.sectionSpace,
+                    const Text(
+                      'Overview',
+                      style: UIConstants.subtitle,
+                    ),
+                    Text(
+                      movie.overview,
+                      style: UIConstants.text,
```

Comment:
I see this style in a couple of text, why did you not create a new component for that, with that constant as a default value

Reply:
done